sys-power/tlp/tlp-1.9.0-r1.ebuild

```
@@ -24,18 +24,16 @@ IUSE="
 
 RDEPEND="
 	dev-lang/perl
-	virtual/udev
-	sys-apps/hdparm
 	net-wireless/iw
-	!app-laptop/laptop-mode-tools
+	sys-apps/hdparm
 	sys-apps/pciutils
-	!sys-apps/tuned
 	sys-apps/usbutils
-	pd? ( dev-python/dbus-python )
-	pd? ( !sys-power/power-profiles-daemon )
-	pd? ( dev-python/pygobject )
-	pd? ( ${PYTHON_DEPS} )
-	pd? ( !sys-apps/tuned )
+	virtual/udev
+	pd? (
+		$(python_gen_cond_dep dev-python/dbus-python[${PYTHON_USEDEP}])
+		$(python_gen_cond_dep dev-python/pygobject[${PYTHON_USEDEP}])
```

```suggestion
		$(python_gen_cond_dep 'dev-python/pygobject[${PYTHON_USEDEP}')
```
You can also combine the calls but it's not required.